frontend/public/models/yaml-templates.ts

```diff
@@ -693,4 +693,11 @@ spec:
     type: JenkinsPipeline
   triggers:
   - type: ConfigChange
+`).setIn([referenceForModel(k8sModels.ClusterServiceBrokerModel), 'default'], `
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ClusterServiceBroker
+metadata:
+  name: example-cluster-service-broker
+spec:
+  url: http://ups-broker-ups-broker.ups-broker.svc.cluster.local
```

Comment:
I know I said to use this URL, but I didn't realize there was nothing else in the example specific to the UPS example broker. So I suggest something like `https://example.com/broker/` as a placeholder. (Sorry.)